backend/src/test/java/gov/cdc/usds/simplereport/db/repository/SupportedDiseaseRepositoryTest.java

```diff
@@ -1,18 +1,32 @@
 package gov.cdc.usds.simplereport.db.repository;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import gov.cdc.usds.simplereport.db.model.SupportedDisease;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.IncorrectResultSizeDataAccessException;
 
 class SupportedDiseaseRepositoryTest extends BaseRepositoryTest {
 
   @Autowired private SupportedDiseaseRepository _repo;
 
   @Test
-  void saveSupportedDisease_successful() {
-    _repo.save(new SupportedDisease("COVID-19", "123"));
-    assertEquals(1, _repo.findAllByName("COVID-19").size());
+  void findSupportedDiseaseByNameContains_successful() {
+    SupportedDisease covid = _repo.findSupportedDiseaseByNameContains("COVID");
+    assertEquals("96741-4", covid.getLoinc());
+  }
+
+  @Test
+  void findSupportedDiseaseByNameContains_needsConclusiveSearchTerm() {
+    Throwable caught =
+        assertThrows(
+            IncorrectResultSizeDataAccessException.class,
+            () -> {
+              _repo.findSupportedDiseaseByNameContains("Flu");
+            });
+    assertTrue(caught.getMessage().contains("query did not return a unique result: 2"));
```

Comment:
non-blocking knowledge question: since we don't write implementation for these methods, are these mostly to test that the repo is in the right state?

Reply:
Yep! These tests ideally verify that the `DbTruncator` changes I made are working correctly, and that the test database loads with the three supported diseases we're expecting without needing to do any setup work in the tests.

The conclusive search test is mostly because I was curious about what error it would throw 😄